README.md

```diff
@@ -1,63 +1,67 @@
-# Tascurator-frontend
+<div align="center">
+	<img src="public/logo.svg" alt="Tascurator Logo" width="120" />
+</div>
 
-## Getting Started
+<br>
 
-First, install the dependencies:
+**Tascurator** is a web application designed to streamline the management of cleaning duties in a share house.
```

Comment:
I think the term `share house` is not that commonly used, so either change the wording or use double quotation is better.